### PR TITLE
Add endpoint to delete a comment

### PR DESCRIPTION
**Pull request by jasonbarrows**

### CODE REVIEW

File: controllers/comments.controllers.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const { deleteCommentById } = require("../models/comments.models");
+
+exports.destroyCommentById = (req, res, next) => {
```
Comment (lewispricey) [inline controllers/comments.controllers.js:3]: Love the function name 😆

File: models/comments.models.js [hunk 2]
```diff
@@ -19,3 +19,17 @@ exports.insertCommentByArticleId = (article_id, { username, body }) => {
...
+exports.deleteCommentById = (comment_id) => {
+  return db.query(
+    `DELETE FROM comments
+    WHERE comment_id = $1
+    RETURNING *;`, [
+      comment_id,
+    ]
+  ).then(({ rows }) => {
+    if (rows.length === 0) {
+      return Promise.reject({ status: 404, message: 'Not Found' });
```
Comment (lewispricey) [inline models/comments.models.js:32]: This is a nice way to handle a none existant ID 👍

**Review by lewispricey — APPROVED** [review]
Great work on this ticket Jason, You've covered all bases with your tests here and the code is very readable thoughout.

Good to merge when you're ready 🚀

File: controllers/comments.controllers.js (hunk 1 above)
Comment (jasonbarrows) [reply controllers/comments.controllers.js:3]: Haha thanks, it's meant to be keeping in line with RESTful controllers and handily it doesn't compete with the name of the model delete method.  😆

**Pull request merged**